src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -26,11 +26,7 @@ def new_patchinfo
     end
 
     read_patchinfo
-    @binaries.each do |bin|
-      if @binarylist.match(bin)
-        @binarylist.delete(bin)
-      end
-    end
+    @binaries.each { |bin| @binarylist.delete(bin) }
```

Comment:
I am not even sure if this could be removed completely. It doesn't seem that the patchinfo OBS creates in this action every has any binaries.